backend/src/model/dto.rs

```diff
@@ -254,6 +266,18 @@ pub struct MapSearchParameters {
     pub is_inactive: Option<bool>,
     /// The owner of the map.
     pub owner_id: Option<i32>,
+    /// Whether or not the map is private.
+    pub privacy: Option<PrivacyOptions>,
+}
+
+/// Support struct for transmitting latitude/longitude coordinates.
+#[typeshare]
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Coordinates {
```

Comment:
You could try to implement `From<Point>` and `Into<Point>` for Coordinates. That way you can replace
```rust
            location: map.location.map(|latlng| Coordinates {
                latitude: latlng.y,
                longitude: latlng.x,
            }),
```
with
```rust
            location: map.location.map(From::from),
```

Same for
```rust
            location: match new_map.location {
                Some(latlng) => Some(Point {
                    x: latlng.longitude,
                    y: latlng.latitude,
                    srid: Some(4326),
                }),
                None => None,
            },
```
could be 
```rust
            location: new_map.location.map(Into::into)
```